docs/release/3.5/release-3-5-46.md

```diff
@@ -0,0 +1,150 @@
+OSG Software Release 3.5.46
+===========================
+
+**Release Date:** 2021-09-09  
+**Supported OS Versions:** EL7, EL8
+
+!!!tip "Want faster access to production-ready software?"
+    OSG 3.5 offers a rolling release repository where packages are added as soon as they pass acceptance testing.
+    To install packages from this repository, enable `[osg-rolling]` in `/etc/yum.repos.d/osg-rolling.repo`:
+
+        [osg-rolling]
+        ...
+        enabled=1
+
+    Or for one-time installations, append the following to your `yum` command:
+
+        --enablerepo=osg-rolling
+
+Summary of Changes
+------------------
+
+!!! note "Important"
+    If you are running an HTCondor-CE with SciTokens and GSI Authentication, you should update to HTCondor 9.0.5
+    at your earliest convenience.
+
+This release contains:
+
+-   Upcoming
+    -   [HTCondor 9.0.5](https://www-auth.cs.wisc.edu/lists/htcondor-world/2021/msg00017.shtml): Bug fix release
+        -   **Other authentication methods are tried if mapping fails using SciTokens**
+        -   Fix rare crashes from successful condor_submit, which caused DAGMan issues
+        -   Fix bug where ExitCode attribute would be suppressed when OnExitHold fired
+        -   condor_who now suppresses spurious warnings coming from netstat
+        -   The online manual now has detailed instructions for installing on MacOS
+        -   Fix bug where misconfigured MIG devices confused condor_gpu_discovery
```

Comment:
Igor takes issue with this statement since the output is still not right, i.e. MIG device still confuse `condor_gpu_discovery`. AIUI, we just fixed the segfaults.

Reply:
A misconfigured MIG GPU would cause no GPUs to be discovered. So, the statement is true. He reported a problem with MIGs and CUDA_VISIBLE_DEVICES (HTCONDOR-620). Release announcement generally do not state what the software does not do. I guess I could add the MIG devices are not supported.

Reply:
How about: Fix bug where misconfigured MIG devices would cause no GPUs to be detected

Reply:
Is that the most obvious outcome of the bug? Or were there stacktraces?

Reply:
No stack traces. Just no GPUs detected. BTW, MIG device support is in the upcoming 9.0.6 release.